organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+The Mission of Hacker Society
+=============================
+
+A time and place for people to learn about technology from each other, in order
+to build a community, to empower the individual, promote equality, and build an
+open society.
+
+**Open Society**: An open society promotes freedom of thought and expression,
+personal responsibility, individualism, cooperation, and equality of all
+citizens.
+
+Code of Conduct
+===============
+
+It is expected that members behave in a way that allows Hacker Society to
+achieve its mission.  Participants in Hacker Society events are expected to
+treat each other with dignity and respect for each other's perspectives,
+experiences, and knowledge.  In particular, participants should recognize that
+people of all skill levels are welcome, and should always be willing to help
+others learn.
+
+To support Hacker Society's mission of: learning technology from each other,
+building community, and promoting equality, Hacker Society must be a place where
+people can be comfortable regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race, age, or
+religion. Therefore, we do not tolerate:
+
+- Rude or unwelcome comments about a person's gender, gender identity, gender
+  expression, sexual orientation, disability, physical appearance, attire, race,
+  age or religious expression.
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, or following
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behavior
+
+Necessarily, the goals of learning, community, and equality are in tension
+with the goals individual empowerment, freedom of thought, and freedom of
+expression. We strive for politeness and civil discourse but we also strive to
+make all welcome. Even those with unpopular ideas or thoughts. However, in
+order to foster our community certain limits on free expression must be
+observed. We expect that debate and constructive criticism are part of this
+open atmosphere. We expect that some members of the community may express
+ideas which are not politically correct. We expect that some ideas expressed
+by a minority may be deeply unpopular and distasteful to the majority.
+However, if these ideas or discussions devolve into attacks against a person
+and/or their intrinsic characteristics, we believe that this is unacceptable.
+
+Enforcement
+-----------
+
+This policy applies at all Hacker Society events and spaces, online and
+offline.  This includes, but is not limited to:
+
+- Weekly talks
+- Open hacks
+- Link-State
+- HackCWRU
+- Mailing lists (hackers-discuss, cwru-hackers)
+- GitHub organization
+- Slack team
```

Comment:
As the dude in charge of the IRC server right now: I do not consent to this policy being applied to IRC because there is already a notice that the server is under the CWRU Acceptable Use Policy.